src/rgw/rgw_sal_motr.cc

```diff
@@ -400,6 +401,9 @@ int MotrUser::store_user(const DoutPrefixProvider* dpp,
     store->store_access_key(dpp, y, MGWUserKeys);
   }
 
+  MotrEmailInfo emailInfo(info.user_email, info.user_id.id);
```

Comment:
Please put an empty check here. We should store anything in email index table only if email is supplied while user creation.